.gitignore

```diff
@@ -138,3 +138,6 @@ dist
 
 # Visual Studio Code
 .vscode
+
+# Vulture output
+dead-code.txt
```

Comment:
Good point. I suppose I was thinking we would do something similar to the dependency script you wrote, but since the data generated from this script isn't fed into another tool, it does feel more like a report. I'll change it